src/main/java/edu/harvard/h2ms/domain/core/Event.java

```diff
@@ -32,29 +32,29 @@
 	@Column
 	private Long id;
 
-	@NotNull
+//	@NotNull
```

Comment:
Why were all these annotations removed? They need to go back

Reply:
ORM was having fits inserting demo data.  Although, this is pretty restrictive constraint - null is missing data.  ID is understandable, but are we rejecting anything but complete data?  What if client can't handle it?

Reply:
An event needs to complete, It’s very typical to have validations to ensure inportant fields are not null, not everything is required but a time stamp certainly is.